docs/reference/treyresearch/armTemplates/portal-es-lite.json

```diff
@@ -322,27 +302,92 @@
                             "link": {
                                 "label": "Learn more",
-                                "uri": "https://docs.microsoft.com/azure/architecture/reference-architectures/hybrid-networking/hub-spoke"
+                                "uri": "https://docs.microsoft.com/azure/security/fundamentals/overview"
                             }
                         }
-                    },                    
+                    },					
+                    {
+                        "name": "esAsc",
+                        "type": "Microsoft.Common.OptionsGroup",
+                        "label": "Deploy Azure Security Center and enable security monitoring for your platform and resources",
+                        "defaultValue": "Yes, Azure Defender On (recommended)",
+                        "toolTip": "If 'Yes' is selected when also adding a subscription for management, ARM will deploy resources and enable them for continous compliance",
+                        "constraints": {
+                            "allowedValues": [
+                                {
+                                    "label": "Yes, Azure Defender On (recommended)",
+                                    "value": "Standard"
+                                },
+                                {
+                                    "label": "Yes, Azure Defender Off",
+                                    "value": "Free"
+                                },
+                                {
+                                    "label": "No",
+                                    "value": "No"
+                                }
+                            ]
+                        },
+						"visible": "[equals(steps('esGoalState').esLogAnalytics, 'Yes')]"
+                    },                 
+                    {
+                        "name": "esSecuritySolution",
+                        "type": "Microsoft.Common.OptionsGroup",
+                        "label": "Deploy Azure Sentinel",
+                        "defaultValue": "No",
+                        "toolTip": "If 'Yes' is selected when also adding a subscription for management, ARM will deploy resources and enable them for continous compliance",
+                        "constraints": {
+                            "allowedValues": [
+                                {
+                                    "label": "Yes",
+                                    "value": "Yes"
+                                },
+                                {
+                                    "label": "No",
+                                    "value": "No"
+                                }
+                            ]
+                        },
+						"visible": "[equals(steps('esGoalState').esLogAnalytics, 'Yes')]"
+                    }					
+                ]
+            },
+            {
+                "name": "esConnectivityGoalState",
```

Comment:
Since a single subscription is being used on the platform side, we cannot separate platform and networking into their own blades while carrying the user specified options made on earlier step. Also, asking for the same subscription twice is not optimal, and selecting different subscription will cause the template to either ignore the subId entirely - or fail

Reply:
I did some testing with the code we are submitting, and it seems to behave as intended even when keeping networking settings in a separate tab. When the user selects the platform subscription in the first tab, the selection of the connectivity subscription is hidden, and the other way around. Anyway, we will do some more testing. We may have been only playing around with the happy path and may be missing something. Thanks for the feedback.

Reply:
@krnese We have been testing this again and we are still not being able to reproduce the error.
https://user-images.githubusercontent.com/8905235/105496597-fb529600-5cbd-11eb-89f0-90e7ff20085c.mp4
Not sure you could please elaborate on the workflow that makes it fail.
Thanks.

Reply:
It also looks good on my side now.